<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanRegister</class>
 <widget class="QWidget" name="LoanRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonExtend">
       <property name="text">
        <string>Extend duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReturn">
       <property name="text">
        <string>Register return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
